### PR TITLE
[tests-only][full-ci] updating lint rules for test code

### PR DESCRIPTION
**Pull request by ghost**
## Description

This PR updated lint rule for test code because there were comments like below.
https://github.com/owncloud/ocis/pull/10660#discussion_r1887871674
its better to add rules for things like that rather then noticing those things in PR review.

Rules added
|rule|description|
|--|---|
|return_type_declaration|https://mlocati.github.io/php-cs-fixer-configurator/#version:2.16\|fixer:return_type_declaration|
|single_space_around_construct|https://cs.symfony.com/doc/rules/language_construct/single_space_after_construct.html#example-1|
|Generic.Commenting.Todo|Warns about TODO comments.|
|Warns about code that can never been executed.|Warns about code that can never been executed.|
|Squiz.WhiteSpace.SuperfluousWhitespace|Checks for unneeded whitespace.|
|Generic.Files.LineLength|Checks no of character in a line|

Ive added few rules which we can removed (`trailing_comma_in_multiline`,`Squiz.WhiteSpace.SuperfluousWhitespace`,..etc) if its too much.

## How Has This Been Tested?

- test environment:
- test case 1: locally
- test case 2:
- ...

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] Technical debt
- [x] Tests only (no source changes)

## Checklist:

- [ ] Code changes
- [ ] Unit tests added
- [ ] Acceptance tests added
- [ ] Documentation ticket raised: <link>

### CODE REVIEW

File: tests/acceptance/TestHelpers/Asserts/WebDav.php [hunk 1]
```diff
@@ -64,7 +64,7 @@ public static function assertDavResponseElementIs(
 		self::assertEquals(
 			$expectedValue,
 			$result,
-			__METHOD__ . " " . $extraErrorText . "Expected '$expectedValue' in element $element got '$result'"
+			__METHOD__ . " " . $extraErrorText . "Expected '$expectedValue' in element $element got '$result'",
```
Comment (amrita-shrestha) [inline tests/acceptance/TestHelpers/Asserts/WebDav.php:67]: do we need this type rules in linter 
```suggestion
			__METHOD__ . " " . $extraErrorText . "Expected '$expectedValue' in element $element got '$result'"
```

**Review by amrita-shrestha — COMMENTED** [review]
add rule to make exception for last line

File: .php-cs-fixer.dist.php [hunk 2]
```diff
@@ -12,6 +12,20 @@
...
+$config->setFinder($finder)
+    ->setIndent("\t")
+    ->setRules(
+        array_merge(
+            $ocRule,
+            [
+                "return_type_declaration" => [
+                    "space_before" => "none",
+                ],
+                'single_space_around_construct' => true
```
Comment (saw-jan) [inline .php-cs-fixer.dist.php:26]: is there anything to break long lines or list of method args/params?

Comment (saw-jan) [reply .php-cs-fixer.dist.php:26]: 120?

**Review by amrita-shrestha — COMMENTED** [review]
lgtm

**Review by saw-jan — APPROVED** [review]

**Review by phil-davis — APPROVED** [review]
Looks OK to me. This will standardize the test code more.

**Pull request merged**